src/bin/blackjack.rs

```diff
@@ -29,14 +35,31 @@ fn set_cargo_path(metadata: &mut MetadataCommand) {
 }
 
 fn main() {
+    let mut pico = pico_args::Arguments::from_env();
```

Comment:
With just two command line arguments I personally don't think we've reached a threshold where a crate to do argument parsing can pull its weight.

I had not heard of `pico_args` before, cool stuff! If we end up needing more arguments later on, I'll probably reach for it 😄